src/rhi-shared.h

```diff
@@ -437,7 +437,7 @@ class SimpleShaderObjectData
 
     Index getCount() { return m_ordinaryData.size(); }
     void setCount(Index count) { m_ordinaryData.resize(count); }
-    uint8_t* getBuffer() { return m_ordinaryData.data(); }
+    uint8_t* getBuffer() { return m_ordinaryData.empty() ? nullptr : m_ordinaryData.data(); }
```

Comment:
probably not needed as we shouldn't access data if the buffer is empty .. but ok to keep too